exporter/zipkinexporter/zipkin.go

```diff
@@ -65,6 +64,15 @@ func createZipkinExporter(cfg *Config) (*zipkinExporter, error) {
 	return ze, nil
 }
 
+// start creates the http client
+func (ze *zipkinExporter) start(_ context.Context, _ component.Host) (err error) {
+	ze.client, err = ze.clientSettings.ToClient()
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
This is equivalent with `return err` or just `return` because you are using named return argument